pkg/webui/components/table/index.js

```diff
@@ -213,6 +227,12 @@ Tabular.propTypes = {
     sortable: PropTypes.bool,
     width: PropTypes.number,
   })),
+  /**
+   * A flag specifying whether the table should paginate entries.
+   * If true the component makes sure that the items are paginated, otherwise
+   * the user is responsible for passing the right number of items.
+   */
+  handlePagination: PropTypes.bool,
```

Comment:
The name makes it sound like a handler function. I'd prefer sth like `handlesPagination` or `clientSidePagination`, but your call.